p4-14/v1.0.3/tex/p4.tex

```diff
@@ -1832,7 +1832,7 @@
 { % Description
 If the \texttt{header_instance} is not an element in a header stack, the indicated 
 header instance is set valid. If the header instance was invalid, all its 
-fields are initialized to 0. If the header instance is already valid, it is 
+fields are initialized to 0 (TBD: Should this be updated to 'all its fields contain unspecified values'?). If the header instance is already valid, it is 
```

Comment:
Both P4-14 and P4-16 specs say that the value of fields in an invalid header are undefined.  I think there is strong motivation from multiple P4 implementers not to change that.